srcpkgs/gst-plugins-base1/template

```diff
@@ -16,27 +16,36 @@ makedepends="gstreamer1-devel glib-devel libxml2-devel pango-devel
  libSM-devel orc-devel $(vopt_if cdparanoia libcdparanoia-devel)
  opus-devel MesaLib-devel $(vopt_if sndio sndio-devel) graphene-devel"
 depends="orc>=0.4.18 gstreamer1>=${version}"
+checkdepends="mesa-dri"
 short_desc="GStreamer Base Plug-ins (v1.x)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later, LGPL-2.0-or-later"
 homepage="https://gstreamer.freedesktop.org"
 distfiles="${homepage}/src/${pkgname/1/}/${pkgname/1/}-${version}.tar.xz"
-checksum=b13e73e2fe74a4166552f9577c3dcb24bed077021b9c7fa600d910ec6987816a
+checksum=dd04fb1f7826e2f6d9b4d66fc22f19cc6a47c301e13041f0ee3d7f65c89b05ac
 
-# Package build options
 build_options="cdparanoia gir sndio"
 build_options_default="cdparanoia gir"
-
-# this should not remain in this package in longer term
-# either upstream, separate, or remove
```

Comment:
why did you partially undo https://github.com/void-linux/void-packages/commit/2e9ed7988c310e872cf22cba88c2886c06a66fd9?

Reply:
Just added additional warning if someone wants to enable sndio since it isn’t to be supported upstream